### PR TITLE
R23.7 notes

### PR DESCRIPTION
**Pull request by adwk67**
Rendered view: https://deploy-preview-420--stackable-docs.netlify.app/home/nightly/release_notes.html

### CODE REVIEW

File: modules/ROOT/pages/release_notes.adoc [hunk 1]
```diff
@@ -3,6 +3,368 @@
...
+== Release 23.7
+
+This release introduces the specification of resource quotas and Pod-overrides and updates the product versions supported by SDP.
+
+=== New / extended platform features
+The following new major platform features were added:
+
+Resource Quotas::
+
+Explicit resources are now applied to all containers, both operators and products. This prevents pre-specified resource limits from being violated. Where these are not specified directly, defaults will be used. See https://github.com/stackabletech/issues/issues/368[this issue] for more information.
```
Comment (sbernauer) [inline modules/ROOT/pages/release_notes.adoc:15]:
```suggestion
Explicit resources are now applied to all containers, for both operators and products. This allows running the Stackable data platform on Kubernetes clusters with a ResourceQuota or LimitRange set. Where these are not specified directly, defaults will be used. See https://github.com/stackabletech/issues/issues/368[this issue] for more information.
```

File: modules/ROOT/pages/release_notes.adoc [hunk 2]
```diff
@@ -3,6 +3,368 @@
...
+
+Pod Overrides::
+
+It is now possible to add custom settings which specify elements of a pod template (service, stateteful set etc.) on roles or rolegroups, which the operator then merges with the objects it writes before actually applying them. See https://github.com/stackabletech/issues/issues/346[this issue] for more details.
```
Comment (sbernauer) [inline modules/ROOT/pages/release_notes.adoc:19]:
```suggestion
It is now possible to add custom settings which specify elements of a pod template (service, stateteful set etc.) on roles or rolegroups, which the operator then merges with the objects it writes before actually applying them. Have a look at https://docs.stackable.tech/home/stable/concepts/overrides.html#_pod_overrides[the documenation] for more details.
```

File: modules/ROOT/pages/release_notes.adoc [hunk 3]
```diff
@@ -3,6 +3,392 @@
...
+=== New / extended platform features
+The following new major platform features were added:
+
+Resource Quotas::
+
+Explicit resources are now applied to all containers, for both operators and products. This allows running the Stackable data platform on Kubernetes clusters with a ResourceQuota or LimitRange set. Where these are not specified directly, defaults will be used. See https://github.com/stackabletech/issues/issues/368[this issue] for more information.
+
+Pod Overrides::
+
+It is now possible to add custom settings which specify elements of a pod template (Service, StatetefulSet etc.) on roles or rolegroups, which the operator then merges with the objects it writes before actually applying them. This provides the user with a possibility for specifying any property that can be set on a regular Kubernetes Pod, but which is not directly exposed via the Stackable custom resource definition. Have a look at xref:concepts:overrides.adoc[the documentation] for more details.
```
Comment (maltesander) [inline modules/ROOT/pages/release_notes.adoc:19]:
```suggestion
It is now possible to add custom settings which specify elements of a pod template (Service, StatefulSet etc.) on roles or rolegroups, which the operator then merges with the objects it writes before actually applying them. This provides the user with a possibility for specifying any property that can be set on a regular Kubernetes Pod, but which is not directly exposed via the Stackable custom resource definition. Have a look at xref:concepts:overrides.adoc[the documentation] for more details.
```

File: modules/ROOT/pages/release_notes.adoc [hunk 4]
```diff
@@ -3,6 +3,392 @@
 The Stackable platform consists of multiple operators that work together.
 Periodically a platform release is made, including all components of the platform at a specific version.
 
+== Release 23.7
+
+This release introduces the specification of resource quotas and Pod-overrides and updates the product versions supported by SDP.
+
+=== New / extended platform features
+The following new major platform features were added:
```
Comment (maltesander) [inline modules/ROOT/pages/release_notes.adoc:11]:
```suggestion
=== New / extended platform features

The following new major platform features were added:
```

File: modules/ROOT/pages/release_notes.adoc [hunk 5]
```diff
@@ -3,6 +3,392 @@
...
+
+Resource Quotas::
+
+Explicit resources are now applied to all containers, for both operators and products. This allows running the Stackable data platform on Kubernetes clusters with a ResourceQuota or LimitRange set. Where these are not specified directly, defaults will be used. See https://github.com/stackabletech/issues/issues/368[this issue] for more information.
+
+Pod Overrides::
```
Comment (maltesander) [inline modules/ROOT/pages/release_notes.adoc:17]: Sometimes Pod overrides or pod-overrides, lets keep this consistent?

**Review by maltesander — COMMENTED** [review]
Just minor things. LGTM!

File: modules/ROOT/pages/release_notes.adoc (hunk 5 above)
Comment (adwk67) [reply modules/ROOT/pages/release_notes.adoc:18]: Done. Changed to `pod overrides` as we use `configuration overrides` (lowercase) elsewhere.

**Review by maltesander — APPROVED** [review]
LGTM!

**Pull request merged**